src/Npgsql/NpgsqlCommandBuilder.cs

```diff
@@ -356,7 +356,8 @@ private static void SetParameterValuesFromRow(NpgsqlCommand command, DataRow row
         /// <param name="whereClause">if set to <c>true</c> [where clause].</param>
         protected override void ApplyParameterInfo(DbParameter p, DataRow row, System.Data.StatementType statementType, bool whereClause)
         {
-            // TODO: We may need to set NpgsqlDbType, as well as other properties, on p
+            var param = (NpgsqlParameter)p;
```

Comment:
nit: unify to a single line and convert to expression-bodied method

Reply:
Ok @roji,  I'll change that.
There are several other instances of one line methods in this class that could be converted to expression-bodied methods (also in [dev](https://github.com/Brar/npgsql/blob/dev/src/Npgsql/NpgsqlCommandBuilder.cs)).
I'd prefer converting them all in a separate commit in dev and backporting this commit if that's ok for you.

Reply:
> I'd prefer converting them all in a separate commit

Oh,
`git diff npgsql/hotfix/3.2.8 dev .\src\Npgsql\NpgsqlCommandBuilder.cs`
just teached me that there were quite a few other style changes between 3.2.7 and dev so this would need two separate style commits for both branches.
I think I'll fix your comment above before merging and afterwards we can discuss if we want do backport other style changes.